### ISSUE TITLE
Incorrect result in IsFactorisableInverseMonoid for an ideal

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#530]
`I` and `J` are equal, but one is a factorisable inverse monoid, and the other is not:
```
gap> S := SymmetricInverseMonoid(5);;
gap> I := MinimalIdeal(S);;
gap> IsFactorisableInverseMonoid(I);
false
gap> J := InverseMonoid(I);;
gap> I = J;
true
gap> IsFactorisableInverseMonoid(J);
true
```

**Comment by james-d-mitchell** [comment]
Yikes, thanks for the report @wilfwilson!

**Comment by james-d-mitchell** [comment]
For info the old method for `IsFactorisableInverseMonoid` for an ideal is:

~~~~
InstallMethod(IsFactorisableInverseMonoid, "for an inverse semigroup ideal",
[IsSemigroupIdeal and IsInverseSemigroup],
function(I)
  if I = SupersemigroupOfIdeal(I) then
    return IsFactorisableInverseMonoid(SupersemigroupOfIdeal(I));
  fi;
  return false;
end);
~~~~

This is clearly incorrect, as your example shows. A fix is in the works.

**Comment by james-d-mitchell** [comment]
Fixed by PR #539

**Comment by james-d-mitchell** [comment]
Resolved in v3.0.20.
